packages/console/tsconfig.json

```diff
@@ -7,23 +7,13 @@
   "include": ["src/**/*.ts", "src/**/*.tsx", "src/**/*.js", "src/**/*.jsx"],
   "exclude": ["node_modules", "src/**/*.test.*", "src/**/__mocks__/*"],
   "references": [
-    {
-      "path": "../components"
-    },
-    {
-      "path": "../jsapi-shim"
-    },
-    {
-      "path": "../log"
-    },
-    {
-      "path": "../storage"
-    },
-    {
-      "path": "../utils"
-    },
-    {
-      "path": "../chart"
-    }
+    { "path": "../components" },
+    { "path": "../jsapi-bootstrap" },
+    { "path": "../jsapi-types" },
+    { "path": "../jsapi-shim" },
+    { "path": "../log" },
+    { "path": "../storage" },
+    { "path": "../utils" },
+    { "path": "../chart" }
```

Comment:
Hm.. I'd put chart at the top for alphabetical, and it also looks like we're missing `"../icons"` here (which we depend on in the package.json). Wonder why we haven't gotten an error with that :thinking: 
We really need to write some tests that make sure this stuff is in sync.